src/PrestaShopBundle/Exception/InvalidVersionException.php

```diff
@@ -23,7 +23,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-namespace PrestaShop\PrestaShop\Core\Foundation\Version\Exception;
+namespace PrestaShopBundle\Exception;
```

Comment:
nope, we dont move this code in the bundle

Reply:
Oups, my bad, miss reading `Core/Foundation/Exception` ><

Reply:
I'll do it, I just wanted to be sure you're ok with it :)